bundles/org.openhab.binding.millheat/src/main/java/org/openhab/binding/millheat/internal/model/Heater.java

```diff
@@ -38,7 +38,7 @@ public Heater(final DeviceDTO dto) {
         name = dto.deviceName;
         macAddress = dto.macAddress;
         heatingActive = dto.heaterFlag;
-        canChangeTemp = dto.holiday;
+        canChangeTemp = dto.canChangeTemp; // Was dto.holiday
```

Comment:
I would drop this comment, since this was a bug and can only lead to confusion. It can be tracked in git history.